src/platform/packages/private/kbn-expression-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/expression-utils
+
+Empty package generated by @kbn/generate
```

Comment:
Should we give a little bit of a description of the package here?

Reply:
just leaving empty for now since the description would just be the package name